imod/mf6/simulation.py

```diff
@@ -999,7 +1000,7 @@ def split(self, submodel_labels: GridDataArray) -> Modflow6Simulation:
 
         new_simulation = imod.mf6.Modflow6Simulation(f"{self.name}_partioned")
         for package_name, package in {**original_packages}.items():
-            new_simulation[package_name] = package
```

Comment:
Might be interesting to add a ``.copy`` method to the Package object.

Reply:
Sure,. I'm used to using copy.deepcopy for everything but  for some users it may be helpful.